docs/developer/order-expiration.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Order Expiration
+---
+
+:::info
+
+expireOrdersAfter was introduced in **Saleor 3.13**
+
+deleteExpiredOrdersAfter was introduced in **Saleor 3.14**
+
+:::
+
+:::caution
+
+This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
+feedback. However, it is still undergoing development and is subject to modifications.
+
+:::
+
+## Introduction
+
+Saleor allows finalizing checkout without payment. This can be achieved in two ways:
+
+- by setting `orderSettings.allowUnpaidOrders` for a given `Channel` to `true` and calling `checkoutComplete` without any payment/transaction object
+- by using the `orderCreateFromCheckout` mutation with first order then payment flow.
+
+If the payment process is canceled by the customer and the order remains unpaid for some time, it is possible to set automatic stock releases and remove the order.
```

Comment:
"and remove the order." - would be good to be more precise here. I mean we first mark the order as expired, then after `deleteExpiredOrdersAfter` we delete it. I think it is worth mentioning that this process is based on two steps: `expire` -> `delete`.

Reply:
(I see that you clarify it more below, but still think it would be worth explicitly writing that the process contains two steps.)